x11-plugins/compiz-plugins-extra/compiz-plugins-extra-0.8.14.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools eutils
+
+DESCRIPTION="Compiz Fusion Window Decorator Extra Plugins"
+HOMEPAGE="https://github.com/compiz-reloaded"
+SRC_URI="https://github.com/compiz-reloaded/${PN}/releases/download/v${PV}/${P}.tar.xz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="libnotify"
+
+RDEPEND="
+	gnome-base/librsvg
+	libnotify? ( x11-libs/libnotify )
```

Comment:
Please move USE-conditional deps below unconditional deps. This is a standard practice to improve readability.